src/crimson/common/errorator.h

```diff
@@ -813,6 +813,10 @@ struct errorator {
     using type = errorator<AllowedErrors...>;
   };
 
+  // get a new errorator by extending current one with another errorator
+  template <class E>
+  using extend_ertr = typename unify<E>::type;
```

Comment:
these helpers looks good to me. @rzarzynski could you help review this change as well?